src/MoBi.Presentation/Presenter/ExpressionProfileInitialConditionsPresenter.cs

```diff
@@ -26,6 +26,11 @@ public class ExpressionProfileInitialConditionsPresenter : BuildingBlockWithInit
          IFormulaToValueFormulaDTOMapper formulaToValueFormulaDTOMapper,
          IDimensionFactory dimensionFactory) : base(view, startValueMapper, initialConditionsTask, msvCreator, context, deleteStartValuePresenter, formulaToValueFormulaDTOMapper, dimensionFactory, isPresentSelectionPresenter, negativeStartValuesAllowedSelectionPresenter)
       {
+         HideDeleteColumn();
+         HideIsPresentView();
+         HideDeleteView();
```

Comment:
I dont't quite get why you implement a mix and match of creating methods to encapsulate the view method or calling the view method directly. 
This is ok but surprising to be inconsistent withing the SAME PR